classes/splitLayout.js

```diff
@@ -16,17 +16,22 @@ module.exports = class SplitLayout extends Layout
 
     GenerateHtml(rootPath)
     {
-        let split = this.splitType === SplitType.HORIZONTAL;
-        let calcedSize = `calc(${this.splitDistance}% - (${Config.HandleWidth} / 2))`;
-        let inverseCalcedSize = `calc((100 - ${this.splitDistance}%) - (${Config.HandleWidth} / 2))`;
-        let size = split ? `height: ${calcedSize}; width: 100%` : `width: ${calcedSize}; height: 100%`;
-        let inverseSize = split ? `height: ${inverseCalcedSize}; width: 100%` : `width: ${inverseCalcedSize}; height: 100%`;
-        let handleWidth = split ? `height: ${Config.HandleWidth}; width: 100%` : `width: ${Config.HandleWidth}; height: 100%;`;
+        let isHorizontalSplit = this.splitType === SplitType.HORIZONTAL;
+        let mainDimension = isHorizontalSplit ? "height" : "width";
+        let subDimension = isHorizontalSplit ? "width" : "height";
+        let mainCalc = `calc(${this.splitDistance}% - (${Config.HandleWidth} / 2))`;
+        let subCalc = `calc(${this.splitDistance}% - (${Config.HandleWidth} / 2))`;
 
-        let split1 = `<div class="panel" style="${size}; ${split ? `` : `float: left`}">${this.layout.GenerateHtml(rootPath)}</div>`;
-        let handle = `<div class="handle" style="${handleWidth}; ${split ? `` : `float: left`}"></div>`;
-        let split2 = `<div class="panel" style="${inverseSize}; ${split ? `` : `float: left`}">${this.layoutB.GenerateHtml(rootPath)}</div>`;
+        let panelAStyles = `${mainDimension}: ${mainCalc}; ${subDimension}: 100%; float: left;`;
+        let panelBStyles = `${mainDimension}: ${subCalc};  ${subDimension}: 100%; float: left;`;
+        let handleStyles = `float: left;`;
```

Comment:
This is WAY more readable and will allow us to add more styles easily. I have a few ideas on how to improve it further. Will play around and figure it out.